dataplane/src/vpc/mod.rs

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Open Network Fabric Authors
```

Comment:
These look reasonable for now, though they'll probably change again when we work with @sergeymatov's protoc file for the config coming from the agent.

Reply:
Note that this PR doesn't change the existing objects. I'm just moving stuff around. So yes, they will likely need some adjustments in any case